app/code/Magento/InventoryImportExport/Model/Import/Serializer/Json.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\InventoryImportExport\Model\Import\Serializer;
+
+use Magento\Framework\Serialize\SerializerInterface;
+
+/**
+ * @inheritdoc
+ */
+class Json implements SerializerInterface, LegacyJsonHelperInterface
+{
+    /**
+     * @var SerializerInterface
+     */
+    private $serializer;
+
+    /**
+     * Json constructor.
+     * @param SerializerInterface $serializer
+     */
+    public function __construct(SerializerInterface $serializer)
+    {
+        $this->serializer = $serializer;
+    }
+
+    /**
+     * Encode the mixed $valueToEncode into the JSON format
+     *
+     * @param mixed $valueToEncode
+     * @return string
+     */
+    public function jsonEncode($valueToEncode)
+    {
+        return $this->serializer->serialize($valueToEncode);
```

Comment:
this method should be implemented proxying call to 
`$this->serialize($valueToEncode)`